src/modules/priority/priority.service.ts

```diff
@@ -77,6 +94,18 @@ export class PriorityService {
       }
       foundPriority.updatedAt = new Date();
 
+      const tokens = await this.userService.getSiteUsersTokens(
```

Comment:
you can move this into a function and then just pass the siteId because youre using this code multiple times in the same file